doctr/models/recognition/sar.py

```diff
@@ -141,7 +141,13 @@ def call(
             # shape (N, rnn_units + 1) -> (N, vocab_size + 1)
             logits = self.output_dense(logits, **kwargs)
             # update symbol with predicted logits for t+1 step
-            symbol = tf.argmax(logits, axis=-1)
+            if labels:
```

Comment:
Let's make this test more robust:
```
if labels is not None:
```
otherwise passing a boolean = True would produce errors